aws/map.jinja

```diff
@@ -0,0 +1,6 @@
+{% set aws = salt['grains.filter_by']({
+    'Default': {
+    	'eips': []
+    }
+}, grain='osfinger', merge=salt['pillar.get']('aws',{}), default='Default') %}
```

Comment:
@niallcreech Is this fore any eip that might have already been set to the the instance?